user-guides/data-farming/data-farming-101.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Make extra dosh with active rewards yield in Data Farming
+---
+
+# Data Farm 101
+
+<figure><img src="../../.gitbook/assets/gif/farming.gif" alt=""><figcaption></figcaption></figure>
+
+Data Farming is the core incentive system inside of Ocean Protocol. It offers different ways for participants to help the protocol grow and succeed.
+
+It's broken down into 2 reward streams:
+1. Passive Rewards
+2. Active Rewards
+
+As the name suggests, users need to do very little to earn Passive Rewards and the program is aimed to onboard as many people as possible to help drive liquidity and conviction into the protocol.
+
+Active Rewards is an evolving incentive program that offers different venues for contributors to curate markets, create algorithms, and active do work to imporve the protocol.
+
+All streams operate permissionlessly and anyone is welcomed to participate.
+
+## Passive Rewards stream
+
+Passive Rewards is the first of the main streams within Data Farming.
+
+Passive Rewards offers a defined amount of rewards for each round that is divided amongst veOCEAN holders based on how much they hold.
+
+The primary objective of the Passive Rewards stream is to provide a low-friction, high-accessibility way for anyone that is holding OCEAN to participate and help the protocol grow.
+
+It is by-design, without the need for anyone to put in any significant effort. You simply need to lock your tokens for a period of time.
+
+To learn more about veOCEAN and the Passive Rewards stream [you can click here](user-guides/data-farming/how-to-farm-veOCEAN.md)
+
+## Active Rewards stream & substreams
+
+Active Rewards is the second of the main streams within Data Farming.
+
+Active Rewards offers a defined amount of rewards for each round and is further divided into multiple substreams.
+
+The primary objective of these substreams is to provide users with various opportunities to actively engage in Data Farming. This approach allows users with different skills and interests to collaborate and contribute their unique abilities towards achieving the core values of Ocean Protocol.
+
+Let's review the Active Reward substreams now to learn more about them, and help you get started with contributing to Ocean Protocol and earning rewards.
+
```

Comment:
Many titles, sections, and descriptions are being duplicated around between the Design/System Docs and How-Tos.

As a reader, it becomes really hard to understand which readme is giving me theory (system design) vs. application (how-to). This is bloating the readme/docs and making it really hard to maintain/reduce the complexity of it.

I recommend the docs /rewards/ should be focused more on the mechanisms/systems (and renamed to /data-farming/) and focused on describing the why Vs. the How-To docs /user-guides/ that give step-by-step instructions and examples so you can follow along.

This should make it easier to manage the docs, reduce complexity/duplication, and provide a structure that is intuitive to readers.

Example: data-farming-101.md (which I focused on completion) rather than deleting it (reducing) so I can make sure the documentation is whole. However, there is a ton of redundancy, in this case against rewards/df-challenges.md

The next steps here is to finish completion, and then focus on reduction/coalescing information